great_expectations/core/usage_statistics/usage_statistics.py

```diff
@@ -336,7 +336,7 @@ def usage_statistics_wrapped_method(*args, **kwargs):
                 time_end: int = int(round(time.time() * 1000))
                 delta_t: int = time_end - time_begin
 
-                handler = get_usage_statistics_handler(args)
+                handler = get_usage_statistics_handler(list(args))
```

Comment:
why is this necessary? don't *args come in the form of a list already?

Reply:
@cdkini PyCharm was issuing wrong type error.